<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Advanced</class>
 <widget class="QWidget" name="Advanced">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="1" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="pingLabel">
         <property name="text">
          <string>Ping</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="pingLabel_2">
         <property name="text">
          <string>Custom command</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="pingLine">
         <property name="text">
          <string>192.168.86.1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="exeLine">
         <property name="text">
          <string>192.168.86.1</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="pingBtn">
         <property name="text">
          <string>Ping</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exeBtn">
         <property name="text">
          <string>Execute</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="5" column="2">
    <widget class="QListWidget" name="traceList"/>
   </item>
   <item row="5" column="1" rowspan="2">
    <widget class="QListWidget" name="ipList"/>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="traceBtn">
     <property name="text">
      <string>Trace Route</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="ipConfigBtn">
     <property name="text">
      <string>IP Config</string>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>ipList</zorder>
  <zorder>traceList</zorder>
  <zorder>traceBtn</zorder>
  <zorder>ipConfigBtn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
